Lil-Peep/src/styles/modules/description-banner.scss

```diff
@@ -0,0 +1,30 @@
+.banner_section {
+  background-repeat: no-repeat;
```

Comment:
Properties should be ordered background-position, background-repeat, background-size